database/src/backend/heed/database.rs

```diff
@@ -165,7 +166,8 @@ impl<T: Table> DatabaseRo<T> for HeedTableRo<'_, T> {
 }
 
 //---------------------------------------------------------------------------------------------------- DatabaseRw Impl
-impl<T: Table> DatabaseRo<T> for HeedTableRw<'_, '_, T> {
+// SAFETY: Same as `HeedTableRo`.
+unsafe impl<T: Table> DatabaseRo<T> for HeedTableRw<'_, '_, T> {
```

Comment:
This is not the same as `HeedTableRo` as write txs are not Send, right?

This is still fine, the comment should probably say that the send bound only applies to read tables